changelogs/unreleased/77334.yml

```diff
@@ -0,0 +1,3 @@
+---
+title: "Move Line: fixed impossible to create a move line when partner have financial discount"
```

Comment:
Can you add in the changelog the fact that you changed the parameters of computeFinancialDiscount(MoveLine moveLine, Move move); please